test/lifecycle_test.dart

```diff
@@ -906,44 +906,84 @@ void sharedLifecycleTests<T extends react.Component>({
         'newState': 'new',
       };
 
-      final Map initialProps = unmodifiableMap({'getInitialState': (_) => initialState});
+      Map initialProps;
+      Map expectedProps;
+      dynamic newContext;
+      Null expectedSnapshot;
+      LifecycleTestHelper component;
 
-      final dynamic newContext = isComponent2 ? null : const {};
+      setUp(() {
+        initialProps = unmodifiableMap({'getInitialState': (_) => initialState});
+        expectedProps = unmodifiableMap(defaultProps, initialProps, emptyChildrenProps);
+        newContext = isComponent2 ? null : const {};
+        expectedSnapshot = null;
```

Comment:
These variables are the same as before; their initialization is just split to be able to put them in a `setUp`